draft-ietf-tls-ctls.md

```diff
@@ -354,15 +247,43 @@ group is listed by the code point name in {{RFC8446}}, Section 4.2.7.
 (e.g., ed25519). This implies a literal "signature_algorithms" extension
 consisting solely of this group.
 
-randomSize (integer):
+random (integer):
 : indicates that the ClientHello.Random and ServerHello.Random values
 are truncated to the given values. When the transcript is
 reconstructed, the Random is padded to the right with 0s and the
 anti-downgrade mechanism in {{RFC8446)}, Section 4.1.3 is disabled.
 IMPORTANT: Using short Random values can lead to potential
-attacks. When Random values are shorter than 8 bytes, PSK-only modes
-MUST NOT be used, and each side MUST use fresh DH ephemerals.
-The Random length MUST be less than or equal to 32 bytes.
+attacks. The Random length MUST be less than or equal to 32 bytes.
+When this element is omitted, the random values are created by hashing 
+the ephemeral public key via the hash algorithm indicated in 
+cipherSuite and truncated to 32 bytes. 
```

Comment:
Don't we want the entire KeyShare extension?

Also, I don't think when it's "omitted" but rather when it's 0. That way, the default is current TLS, which seems like a good practice.

Reply:
I removed the part about creating the Random values via hashing of the DH public keys. We can add that again later